GameOfLifeConsole/AppUserInterface.cs

```diff
@@ -0,0 +1,86 @@
+﻿
+namespace GameOfLifeConsole
+{
+    public static class AppUserInterface
+    {
+        public static void Start()
+        {
+            Console.WriteLine("App Title: Game Of Life");
+            Console.WriteLine("Please, select if you would like to see the Blinker life circle (M) or " +
+                "LifeCircle with Random numbers (R). \t");
+            Console.Write("Select : M or R:\t ");   
+        }
+        
+        /// <summary>
+        /// UI visual part method for asking to input the quantity of rows and columns.
+        /// </summary>
+        /// <param name="promptMessage"></param>
+        /// <returns> The visual part the message-request.</returns>
+        public static int RequestNumberFromUser(string promptMessage)
+        {
+            while (true)
```

Comment:
This while true is kinda weird.
I'd do it like 
```
while(!int.TryParse(Console.ReadLine(), out int result))
{
// Here goes exception handling/ message printing  when input was wrong.
}
return result
````

Reply:
I got it. But if it could be possible to leave as it is? in this variant it shows me the error.